CMakeLists.txt

```diff
@@ -233,6 +239,9 @@ endif()
 if(SINGULARITY_BUILD_CLOSURE)
   target_compile_definitions(singularity-eos PRIVATE SINGULARITY_BUILD_CLOSURE)
 endif()
+if(SINGULARITY_USE_HELMHOLTZ)
```

Comment:
you should add something like

```cmake
add_custom_command(
  TARGET singularity-eos
  PRE_BUILD
  COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_SOURCE_DIR}/data
          $<TARGET_FILE_DIR:singularity-eos>/data)
```
which will copy the data directory to the build directory.

Also will need to add similar in `cmake/install.cmake` for installing

Reply:
Ah, I see this was done in the test directory. Nevermind.

Reply:
Still need to add a step to install the data. Something like

```cmake
# make sure `include(GNUInstallDirs)` was done beforehand - should be already
install(DIRECTORY ${PROJECT_SOURCE_DIR}/data DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/singularity-eos)
#NB this usually means `<install-prefix>/share`
```

Reply:
Can't hurt to document the location of the file.